### PR TITLE
connect-popup-refactor [2/3] (passphrase and error screen)

### PR DESCRIPTION
**Pull request by mroz22**
This is still draft. Some parts of code will move to separate PRs

- [x] depends for https://github.com/trezor/trezor-suite/pull/5768
- [x] depends for https://github.com/trezor/trezor-suite/pull/5876
- [x] depends on #5887 
- [x] depends on https://github.com/trezor/trezor-suite/pull/6062 
- [x] depends on https://github.com/trezor/trezor-suite/pull/6069
- [x] depends on #6160
- [x] solve sharing components that use `<Translation>` component
- [x] adding proper tests depend on #6287 
- [x] Collapsible box PR #6316

summary:
 
- the first commit moves Passphrase subcomponets component into @trezor/components so that I can reuse it. Looking forward to any ideas how to do it better. But I would like to proceed with this PR rather sooner than later. cc @tomasklim @dahaca  
- the second commit moves CollapsibleBox component into @trezor/components. I am going to need that in connect-ui. 
- the third commit adds some new icons in @trezor/components
- the fourth commit splits types in @trezor/connect. I am not very happy about this one, but I wasn't able to make typescript work when trying to create strongly typed interface later in @trezor/connect-ui package. Any typescript magician around to discuss that? 
- the fifth commit turns standalone connect-ui components into something I would call connect-ui app. it adds also support for future stuff such as theming or internationalization
- the other commits are refactoring old connect-popup into connect-ui without any changes except for commits: 
  - feat(connect-ui): add error screen which adds timeout to loading of popup and prevents popup closing when there is an error or timeout is reached
  - feat(connect-ui): add Passphrase view which implements suite-like passphrase flow

Issues: 
- solves #4880
- related #4878 
- related #4877 
- maybe related #6129 
- maybe related #5598
- maybe related #5252
- maybe related #4962 
- maybe related #4888
- maybe related #4363
- maybe related #4861 
- maybe related #4857

### CODE REVIEW

File: packages/components/src/components/Passphrase/PassphraseTypeCard.tsx [hunk 1]
```diff
@@ -9,19 +12,19 @@ import {
...
 import { countBytesInString } from '@trezor/utils';
-import { OpenGuideFromTooltip } from '@guide-components';
-import PasswordStrengthIndicator from '@suite-components/PasswordStrengthIndicator';
-import { useTranslation } from '@suite-hooks';
 import { isAndroid } from '@trezor/env-utils';
 
-const Wrapper = styled.div<Pick<Props, 'type' | 'singleColModal'>>`
+import PasswordStrengthIndicator from './PasswordStrengthIndicator';
+
+const MAX_LENGTH = 50;
```
Comment (mroz22) [inline packages/components/src/components/Passphrase/PassphraseTypeCard.tsx:25]: share somehow https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/constants/suite/inputs.ts#L4 ?

File: packages/components/src/components/Passphrase/PassphraseTypeCard.tsx [hunk 2]
```diff
@@ -169,13 +172,17 @@ type Props = {
...
     recreateWallet?: () => void;
+    learnMoreTooltipOnClick?: TooltipProps['guideAnchor'];
+    learnMoreTooltipAppendTo?: TooltipProps['appendTo'];
 };
 
 const DOT = '●';
 
-const PassphraseTypeCard = (props: Props) => {
+export const PassphraseTypeCard = (props: PassphraseTypeCardProps) => {
+    console.log('PassphraseTypeCard', props);
```
Comment (karliatto) [inline packages/components/src/components/Passphrase/PassphraseTypeCard.tsx:182]: Do we need this log here?

**Review by karliatto — APPROVED** [review]
I just added a comment for a log that looks like forgotten.

File: packages/components/src/components/Passphrase/PassphraseTypeCard.tsx [hunk 3]
```diff
@@ -292,15 +299,20 @@ const PassphraseTypeCard = (props: Props) => {
...
                                     <Tooltip
-                                        title={<Translation id="TR_WHAT_IS_PASSPHRASE" />}
-                                        guideAnchor={instance => (
-                                            <OpenGuideFromTooltip
-                                                dataTest="@tooltip/guideAnchor"
-                                                id="/security/passphrase.md"
-                                                instance={instance}
+                                        appendTo={props.learnMoreTooltipAppendTo}
+                                        title={
+                                            <FormattedMessage
```
Comment (mroz22) [inline packages/components/src/components/Passphrase/PassphraseTypeCard.tsx:304]: todo: issue, comment (backport, etc.)

**Pull request merged**